### ISSUE TITLE
Inquiry about BTCV Results

### ISSUE THREAD

**Issue by Ljh-11** [Luffy03/VoCo#34]
Hi, thank you for your excellent work. I have a few questions:

Are the BTCV results in Table 1 of the paper based on the validation set? I reproduced the Swin-UNETR "from scratch" results from Table 1 using /VoCo/Finetune/BTCV/main.py and found that the validation accuracy (val_acc) can already reach 82.61 at just 1k iterations. Therefore, I am confused about why there is such a significant discrepancy compared to the reported results.

The results recorded in Table 6 of the Supplementary Materials appear to be from the online test set. Why are the results for Swin-UNETR and VoCo so much higher than those in Table 1 of the main text?
